spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -238,6 +238,27 @@ def update_price_tier!(price_tier)
       def price_tier
         client.price_tier(self.apple_id)
       end
+      
+      #####################################################
+      # @!group purchases
+      #####################################################
+      # Get a List of IAP's
+      def purchases
```

Comment:
Could we rename this maybe? Otherwise it might sound like the number of list of purchases/downloads of the app. What name does Apple use?